v_4_0_0/render_asset_content.go

```diff
@@ -21,3 +22,19 @@ func RenderAssetContent(assetContent string, params interface{}) ([]string, erro
 	content := strings.Split(buf.String(), "\n")
 	return content, nil
 }
+
+func RenderFileAssetContent(assetContent string, params interface{}) (string, error) {
```

Comment:
This function returns base64 representation of the assetContent and used in operators for generating files with ignition spec.

Reply:
Does `RenderIngnitionFileContent` name make sense here? Also you can write your comment a godoc.

Reply:
I still think the name makes sense then, but I leave it to you.